acs-integration-tests/uaa/config/uaa.yml

```diff
@@ -1,8 +1,8 @@
-# Configuration in this file is overridden by an external file
+# Configuration in this file ixs overridden by an external file
```

Comment:
Align the version of uaa.yml with the the version of CF UAA we decide to use - latest or the one supported by predix-uaa.